cache_descriptor.go

```diff
@@ -39,62 +39,69 @@ func (r result) hasChanges() bool {
 }
 
 // compare compares two cache descriptor file and return the differences.
-func compare(old map[string]string, new map[string]string) (r result) {
+func compare(old map[string]string, new map[string]string) result {
```

Comment:
why did we do this change?

Reply:
it is not recommended (linters also call out) long functions that naked return values.